.github/workflows/perf-test-eks.yaml

```diff
@@ -6,6 +6,16 @@ on:
         description: "Mesh version"
         required: true
         type: string
+      num_services:
+        description: "Number of services to run during testing"
+        required: false
+        type: number
+        default: 5
+      instances_per_service:
+        description: "Number of instances per service to run during testing"
+        required: false
+        type: number
+        default: 1
```

Comment:
why have the defaults here? doesn't this make it more difficult to run locally?

Reply:
I'll remove this after merging #52